### PR TITLE
docs: Added Apcera to awesome-minio.

### PR DESCRIPTION
**Pull request by koolhead17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ List of projects bundling Minio server in it's stack.
 
 ### Awesome Use Cases
 List of projects using Minio server to store their data.
-* [www.houbank.com](https://www.houbank.com) - www.houbank.com uses Minio as a simple, reliable and compatible self-host AWS S3 alternative. Houbank uses Minio to upload user files and exchange documents in internal applications/jobs.
+* [Apcera](https://www.apcera.com/)- Apcera is a container management platform for cloud-native and legacy applications. Apcera uses Minio as its backend object storage server.
```
Comment (harshavardhana) [inline README.md]: This is not correct..

Comment (harshavardhana) [inline README.md:20]: They are providing minio as a service,  i don't know if we can add them as using Minio as backend.

Comment (koolhead17) [reply README.md:20]: Adding Minio part of their object storage stack will be better?

Comment (koolhead17) [reply README.md:20]: @GarimaKapoor feedback please.

Comment (deekoder) [reply README.md:20]: Needs more info on how Apcera uses Minio.

**Comment by koolhead17** [comment]
closing this Nitish will send PR with detailed description.

**Comment by GarimaKapoor** [comment]
@koolhead17  This issue is not closed. We have to keep it open till it is fixed.

**Comment by GarimaKapoor** [comment]
@NitishT can you take a look at this, @koolhead17 can share details with you.

**Comment by nitisht** [comment]
sure @GarimaKapoor. @koolhead17 gave some info already, with check with him for more.

File: README.md (hunk 1 above)
Comment (nitisht) [reply README.md:20]: Apcera offers Minio as a managed service to its customers/users. I think we can add a separate category -> Managed services.

https://docs.apcera.com/services/types/service-minio/

**Comment by harshavardhana** [comment]
Closing this PR as merged.

**Pull request closed without merging**